### PR TITLE
refactor(action-bar, action-pad, action-group): Remove usage of getSlotted utility

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #6059

## Summary

- Depends on #7435

### CODE REVIEW

File: packages/calcite-components/src/components/action-bar/action-bar.e2e.ts [hunk 1]
```diff
@@ -165,10 +165,12 @@ describe("calcite-action-bar", () => {
       const page = await newE2EPage();
 
       await page.setContent(html`<calcite-action-bar expand-disabled></calcite-action-bar>`);
+      await page.waitForChanges();
 
       const buttonGroup = await page.find(`calcite-action-bar >>> .${CSS.actionGroupEnd}`);
 
-      expect(buttonGroup).toBeNull();
+      expect(buttonGroup.getAttribute("hidden")).toBe("");
```
Comment (jcfranco) [inline packages/calcite-components/src/components/action-bar/action-bar.e2e.ts:172]: For consistency, can you check the prop instead?

File: packages/calcite-components/src/components/action-bar/action-bar.tsx [hunk 2]
```diff
@@ -345,6 +351,18 @@ export class ActionBar
     this.setGroupLayout(groups);
   };
 
+  handleActionsEndSlotChange = (event: Event): void => {
+    this.hasActionsEnd = slotChangeHasAssignedElement(event);
+  };
+
+  handleTooltipSlotChange = (event: Event): void => {
+    const tooltips = slotChangeGetAssignedElements(event).filter((el) =>
+      el?.matches("calcite-tooltip")
```
Comment (jcfranco) [inline packages/calcite-components/src/components/action-bar/action-bar.tsx:360]: Just curious, when would this be `null`?

**Review by jcfranco — APPROVED** [review]
This LGTM and also looks low-risk.

File: packages/calcite-components/src/components/action-bar/action-bar.tsx (hunk 2 above)
Comment (driskull) [reply packages/calcite-components/src/components/action-bar/action-bar.tsx:360]: are you talking about `el`? If there are no assigned elements.

File: packages/calcite-components/src/components/action-bar/action-bar.e2e.ts (hunk 1 above)
Comment (driskull) [reply packages/calcite-components/src/components/action-bar/action-bar.e2e.ts:172]: @jcfranco I can't do this I don't think?

image

Comment (driskull) [reply packages/calcite-components/src/components/action-bar/action-bar.e2e.ts:172]: Can't use getProperty as its not a component.

image

File: packages/calcite-components/src/components/action-bar/action-bar.tsx (hunk 2 above)
Comment (jcfranco) [reply packages/calcite-components/src/components/action-bar/action-bar.tsx:360]: > If there are no assigned elements.

If it returns an empty array for this case, the `filter` callback won't be invoked.

File: packages/calcite-components/src/components/action-bar/action-bar.e2e.ts (hunk 1 above)
Comment (jcfranco) [reply packages/calcite-components/src/components/action-bar/action-bar.e2e.ts:172]: Maybe it's a doc thing, but we already do this in [`tab-title.e2e.ts`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden).

Comment (driskull) [reply packages/calcite-components/src/components/action-bar/action-bar.e2e.ts]: yeah must be a doc thing.

**Pull request merged**